### ISSUE TITLE
Questions regarding microtask-6

### ISSUE THREAD

**Issue by vchrombie** [vchrombie/chaoss-microtasks#8]
Feel free to drop a comment if you have any question regarding the [microtask-6](https://github.com/vchrombie/chaoss-microtasks/tree/master/microtask-6).

I will be happy to answer you back and take your suggestions. :slightly_smiling_face:

**Comment by valeriocos** [comment]
Thanks @vchrombie , the microtask looks good. Did you create the query manually? I'm asking this because there is a trick you can use. You create a table visualization representing the query you want to do, then you check the request created by Kibana :)

![captura_239](https://user-images.githubusercontent.com/6515067/75873879-01732600-5e11-11ea-8340-ee0037a239ca.png)

**Comment by vchrombie** [comment]
> Thanks @vchrombie , the microtask looks good. 

Thank You @valeriocos. :slightly_smiling_face: 

> Did you create the query manually? I'm asking this because there is a trick you can use. You create a table visualization representing the query you want to do, then you check the request created by Kibana :)

No, I knew the trick already. :grimacing: 

Initially, it was hard to frame the query but I was lurking around, trying different visualizations and experimenting and then I found out this option in the dashboard which made the work really easy. I selected the git authors to tag cloud visualization and got the Request. 

![image](https://user-images.githubusercontent.com/25265451/75882618-89742280-5e47-11ea-925d-b75b1f23bafa.png)

I tried experimenting with the query so that I would understand how the query works. :smile:
